### PR TITLE
ImageMagick: Update to 6.9.13-16 and 7.1.1-38

### PR DESCRIPTION
**Pull request by eschnett**

### CODE REVIEW

File: I/ImageMagick/ImageMagick@6/build_tarballs.jl [hunk 1]
```diff
@@ -61,7 +61,7 @@ dependencies = [
     Dependency("Zlib_jll"),
     Dependency("libpng_jll"),
     Dependency("JpegTurbo_jll"),
-    Dependency("Libtiff_jll"; compat="4.5.1"),
+    Dependency("Libtiff_jll"; compat="4.3.0"),
```
Comment (giordano) [inline I/ImageMagick/ImageMagick@6/build_tarballs.jl:64]: #3024
```suggestion
    Dependency("Libtiff_jll"; compat="~4.5.1"),
```

**Review by giordano — DISMISSED** [review]

File: I/ImageMagick/ImageMagick@6/build_tarballs.jl (hunk 1 above)
Comment (eschnett) [reply I/ImageMagick/ImageMagick@6/build_tarballs.jl:64]: This is for the old (version 6) ImageMagick. Should I do the same for the new version (7)?

Comment (eschnett) [reply I/ImageMagick/ImageMagick@6/build_tarballs.jl]: Done.

**Comment by eschnett** [comment]
The build fails for `x86_64-linux-gnu-cxx03` with
```
[14:00:25] /workspace/destdir/lib/libopenjp2.so: undefined reference to `memcpy@GLIBC_2.14'
```
This is triggered by the change to the newer Libtiff (from 4.3.0 to 4.5.1).

**Comment by eschnett** [comment]
Switching to GCC 6 (same as Libtiff) resolved the problem.

**Comment by ViralBShah** [comment]
Merge?

**Comment by eschnett** [comment]
Yes please.

**Comment by ViralBShah** [comment]
@eschnett I believe you have merge rights...

**Pull request merged**